src/basics.md

```diff
@@ -470,6 +472,50 @@ fn main() {
 }
 ```
 
+[ex-extract-hashtags]: #ex-extract-hashtags
+<a name="ex-extract-hashtags"></a>
+## Extract a list of unique #Hashtags from a text
+
+[![regex-badge]][regex] [![cat-text-processing-badge]][cat-text-processing]
+
+Extracts a sorted and deduplicated list of hashtags from a text.
+
+The hashtag regex given here only catches latin hashtags that start with a letter. The [twitter hashtag regex] is way more complicated.
```

Comment:
I would add some adjective here 
> The [twitter hashtag regex] is way more complicated.

Like:
The complete [twitter hashtag regex] is way more complicated.